### PR TITLE
Add workflow for quickstart validation via devcontainer

### PR DESCRIPTION
**Pull request by CodeMonkeyLeet**
# Description

Create a validation workflow using the .devcontainer description and most of the runtime properties associated with the default devcontainer.json config.

## Issue reference

N/A. Currently exploratory. **#donotmerge**

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] The quickstart code compiles correctly
* [ ] You've tested new builds of the quickstart if you changed quickstart code
* [ ] You've updated the quickstart's README if necessary

### CODE REVIEW

**Comment by CodeMonkeyLeet** [comment]
@wcs1only Based on your suggestion from the PR for codespaces; I have it broken out as a separate workflow right now since:
 - it cannot accommodate the macos (or windows) testing as well as desired in #417 
 - it doesn't support multi-node K8s cluster configuration on minikube in the devcontainer, as is configured for KIND in the existing workflow.

File: .github/workflows/validate-devcontainer.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    branches:
+      - master
+      - release-*
+  workflow_dispatch:
+jobs:
+  test_devcontainer:
+    name: Validate ${{ matrix.quickstart }}/README.md on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    env:
+      DAPR_CLI_VERSION: 1.2.0
```
Comment (wcs1only) [inline .github/workflows/validate-devcontainer.yaml:26]:
```suggestion
DAPR_CLI_VERSION: 1.6.0-rc.1
```

File: .github/workflows/validate-devcontainer.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      DAPR_RUNTIME_VERSION: 1.2.0
```
Comment (wcs1only) [inline .github/workflows/validate-devcontainer.yaml:27]: DAPR_RUNTIME_VERSION: 1.6.0-rc.3

File: .github/workflows/validate-devcontainer.yaml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    branches:
+      - master
+      - release-*
+  workflow_dispatch:
+jobs:
+  test_devcontainer:
+    name: Validate ${{ matrix.quickstart }}/README.md on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    env:
+DAPR_CLI_VERSION: 1.6.0-rc.1
```
Comment (wcs1only) [inline .github/workflows/validate-devcontainer.yaml:26]:
```suggestion
      DAPR_CLI_VERSION: 1.6.0-rc.1
```

File: .github/workflows/validate-devcontainer.yaml [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+DAPR_RUNTIME_VERSION: 1.6.0-rc.3
```
Comment (wcs1only) [inline .github/workflows/validate-devcontainer.yaml:27]:
```suggestion
      DAPR_RUNTIME_VERSION: 1.6.0-rc.3
```

**Comment by amulyavarote** [comment]
We have a different quickstarts structure and content. Hence closing this PR.

**Pull request closed without merging**